solidity/deploy/00_deploy_light_relay.ts

```diff
@@ -0,0 +1,32 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  // LightRelay is deployed only for mainnet. For all other networks (Hardhat,
+  // Goerli) we use TestRelay contract. LightRelay will work properly only with
+  // Bitcoin Mainnet headers.
+  const lightRelay = await deployments.deploy("LightRelay", {
+    contract: hre.network.name === "mainnet" ? "LightRelay" : "TestRelay",
+    from: deployer,
+    log: true,
+    waitConfirmations: 1,
+  })
+
+  if (hre.network.tags.etherscan) {
+    await helpers.etherscan.verify(lightRelay)
+  }
+
+  if (hre.network.tags.tenderly) {
+    await hre.tenderly.verify({
+      name: "LightRelay",
+      address: lightRelay.address,
+    })
+  }
```

Comment:
We should also add these steps to `01_deploy_tbtc_v2_token.ts` and `02_deploy_vending_machine.ts`.

Reply:
See 697ce84770d002b1737e8704ba444299339680a7 and 70bd0562b1aaa43330a6d03afa8b631b03a71f95, please.